app/scripts/controllers/client/ViewClientController.js

```diff
@@ -17,6 +17,8 @@
             scope.updateDefaultSavings = false;
             scope.charges = [];
 
+            scope.collaterals = [];
```

Comment:
Is this array required later in the file?

Reply:
Yes. In line 179,180,